tests/rptest/tests/partition_movement_upgrade_test.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, test_context):
         self._stop_move = threading.Event()
 
     def setUp(self):
-        self.installer.install(self.redpanda.nodes, (22, 1, 5))
+        self.old_version, self.old_version_str = self.installer.install(
```

Comment:
This test isn't specific to the 22.1->22.2 hop: it should upgrade from previous feature release to HEAD

Reply:
done